wifiphisher/common/tui.py

```diff
@@ -281,9 +284,9 @@ def display_info(self, screen, templates, template_names):
             # display the words of chosen template
             if key == ord("\n"):
                 try:
-                    screen.addstr(row_number, 3, "YOU HAVE SELECTED " +
-                                  template_names[self.heightlight_number],
-                                  curses.A_BOLD)
+                    screen.addstr(
+                        row_number, 3, "YOU HAVE SELECTED " +
+                        template_names[self.heightlight_number], curses.A_BOLD)
```

Comment:
It's "highlight" not "heightlight".

Reply:
I know but that is the variable used. The typo was made probably during `tui` creation. We can change that in another PR if you want.